kolibri/core/assets/src/vue/core-base.vue

```diff
@@ -63,6 +63,7 @@
     overflow-y: scroll
     height: 100%
     width: 100%
+    padding-top: $right-margin
```

Comment:
Right margin for padding top? :) that doesn't seem right...

If possible it should be defined in terms of the height of the top bar